common/openapi.yml

```diff
@@ -832,10 +862,15 @@ components:
     PipelineOutputDefinition:
       description: |
         An output field and its specifications for the Pipeline.
+      required: [ name, displayName, type ]
```

Comment:
adding these required fields so that the Python client knows they'll be defined and we don't have to do dumb things for type agreement